src/identity/components/Name.test.tsx

```diff
@@ -43,11 +44,13 @@ describe('OnchainAddress', () => {
     (useIdentityContext as jest.Mock).mockReturnValue({
       schemaId: '0x123',
     });
+    const restore = silenceError();
```

Comment:
In jest test, if a test is expected to throw add these in between:
```
const restore = silenceError();
expect(...).toThrow(...);
restore();
```

This will makes the false console.error go away in test results cc @Zizzamia @abcrane123 @cpcramer @ilikesymmetry

Reply:
nice